package.json

```diff
@@ -25,6 +25,8 @@
   "homepage": "https://github.com/aburd/sci-fi-books#readme",
   "dependencies": {
     "cli-select": "^1.1.2",
+    "dotenv": "^16.0.0",
+    "lmdb": "^2.3.6",
```

Comment:
Our "in-memory" DB.
Basically, this database runs in the same process as our program.
We don't have to start this database separately and manage it.